examples.json

```
@@ -397,6 +397,24 @@
         "Examples > DevOps > Infrastructure as Code > Terraform"
       ],
       "labels": ["Terraform", "AWS", "Minecraft", "Docker", "EC2"]
+    },
+    {
+      "name": "AWS Cognito Local Example",
+      "description": "AWS Cognito local enviroment with Docker and Serverless offline",
+      "url": "https://github.com/nanlabs/devops-reference/tree/main/examples/serverless-cognito-local/",
+      "tags": [
+        "Examples > DevOps > Serverless Framework > Containers"
```

```suggestion
        "Examples > DevOps > Infrastructure as Code > Serverless Framework, SAM and CloudFormation"
```

this string is being use to generate the readme, so we want to keep it consistent with the rest of the examples in the same category